### ISSUE TITLE
Maximum Downloads reached for Ablation Study Data

### ISSUE THREAD

**Issue by zwxlyy1995** [lukaskondmann/DENETHOR#6]
Hi, Lukas

Bother you again.

The download link of Ablation Study Data has reached the maximum number of downloads. See following:
![image](a.assets/191447943-5ec931bd-f4e7-455d-8e50-32b0ea238859.png)

Do you have the management privileges of increasing the maximum limitation of this link ? 

Thanks.

**Comment by zwxlyy1995** [comment]
Oh, sry, The picture url is not correct. See follwing:
![image](https://user-images.githubusercontent.com/45357451/191450514-d77fe93f-9925-4566-ad38-25ff3aecf71e.png)

**Comment by lukaskondmann** [comment]
Hi, I looked into this, and we indeed seem to have reached some traffic quota at LRZ. It's not only related to the number of downloads but also to the general traffic that was apparently too high. I will try to find a solution to this. Potentially, I'll have to move the data to a different source. For now, the download of the ablation data is disabled, unfortunately. I adjusted the readme accordingly
